### PR TITLE
Web4660 move to GitHub

### PR DESCRIPTION
**Pull request by kevinhodges**
#### What are the links to relevant tickets?

[https://hxshortbreaks.atlassian.net/browse/WEB-4660](WEB-4660)
#### What does this PR do?

Moves loggly-interface module to Github.
#### What functional/unit tests does this PR have?

Coverage of the constructor with 3 unit tests of the log function.
#### How should a developer review this?

Full code review and run the tests.
#### Any background context you want to provide?

N/A
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.
#### Reviewer 1
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!
#### Reviewer 2 Emily
- [X] I agree with the assumptions made in the quality checklist.
- [X] I’ve witnessed the work behaving as expected.
- [X] I’ve witnessed the tests running successfully.
- [X] I’ve checked for appropriate test coverage.
- [X] I’ve checked for coding anti-patterns.
- [X] I've checked this work against the requirements of the Jira.
- [X] I don’t believe this work introduces unnecessary technical debt.
- [X] +1 from me!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
+    "url": "git@github.com:holidayextras/loggly-interface.git"
+  },
+  "license": "MIT",
+  "main": "index",
+  "engines": {
+    "node": ">= 0.10.0"
+  },
+  "scripts": {
+    "start": "node launch.js",
+    "test": "istanbul cover _mocha -- --recursive test/ -R spec"
```
Comment (lukefrake) [inline package.json]: Should this be - `grunt jshint && grunt jscs && istanbul cover _mocha -- test/ -R spec`

Comment (kevinhodges) [inline package.json]: Yes, i've updated the call and Rojo is going to update Fillmore

File: README.md [hunk 2]
```diff
@@ -34,13 +29,10 @@ type and namespace
...
-Code is linted by ".jshintrc" and checked against the coding style guide "shortbreaks.jscs.json" when you run the default grunt task:
-```
-$ grunt
-```
+Code is linted checked against the style guide with [make-up](https://github.com/holidayextras/make-up), running npm test will run all tests required.
 
 ## License
-Copyright © 2015 Shortbreaks
-Licensed under the MIT license.
+Copyright (c) 2015 Shortbreaks
```
Comment (lukefrake) [inline README.md]: (c) doesn't turn to © on Github, so should be ©

**Comment by lukefrake** [comment]
1 comment about the copy write symbol, but other than that It's all good.

**Comment by lukefrake** [comment]
I can't :+1: on the comment, I have no idea why, but it's all good!

**Comment by panicnot** [comment]
Can't edit either, but all runs. +1 for me

**Pull request merged**